crawler/plugins_manager.py

```diff
@@ -48,12 +51,11 @@ def _load_plugins(
         category_filter={},
         filter_func=lambda *arg: True,
         features=['os', 'cpu'],
+        plugin_places=['plugins'],
```

Comment:
ah I get it, makes sens now that 'plugin_places' is no longer stored in 'options'